src/Jira/Api.php

```diff
@@ -72,6 +72,9 @@ public function __construct(
         AuthenticationInterface $authentication,
         ClientInterface $client = null
     ) {
+        //Regular expression to remove trailing slash
+        $endpoint = preg_replace('{/$}', '', $endpoint);
```

Comment:
By replace code with solution below no explanatory comment is needed:

```php
$endpoint = rtrim($endpoint, '/');
```

Sounds like useful change, but not sure how it's related to this PR.